src/components/ChatAutoComplete/ChatAutoComplete.js

```diff
@@ -147,24 +147,26 @@ const ChatAutoComplete = (props) => {
               return [];
             }
             const selectedCommands = commands.filter(
-              (c) => c.name.indexOf(q) !== -1,
+              (c) => c.name?.indexOf(q) !== -1,
```

Comment:
commands should always have a name right?

Reply:
This is the question I was posing in Slack, we need to confirm with the backend, will stay conservative for now.